app/javascript/packages/document-capture/context/app.js

```diff
@@ -4,9 +4,12 @@ import { createContext } from 'react';
  * @typedef AppContext
  *
  * @prop {string} appName name of the application (probably Login.gov)
+ * @prop {string} arcgisSearchEnabled feature flag for enabling address search
```

Comment:
Can we type this as a `boolean` and coerce it to the expected type as part of the initialization logic in `packs/document-capture.tsx` ?

Reply:
I see that we _are_ coercing it to a boolean already, we just need to update the type accordingly.

(Ideally TypeScript would have caught this for us, but `composeComponents` isn't type-checking correctly at the moment)